scripts/plop/templates/component/index.tsx.hbs

```diff
@@ -0,0 +1,3 @@
+import { {{pascalCase name}}, {{pascalCase name}}Props } from './{{pascalCase name}}';
```

Comment:
We might need to add isolated modules in our tsconfig. Should protect us from importing things that get stripped at build time.

What that means is the `{{pascalCase name}}Props` will not be valid. So it should be removed.

Reply:
This is the case with our latest `tsconfig`, should we update?

Reply:
Yes